src/components/Explorer/Explorer.tsx

```diff
@@ -138,6 +146,33 @@ const Explorer = () => {
         return null;
     };
 
+    const journeyParams1: JourneyParams = {
+        fromStationId: "place-NEC-2173",
+        toStationId: "place-bbsta",
+        day: "weekday",
+        time: 32010,
+    };
+
+    const journeyParams2: JourneyParams = {
+        fromStationId: "place-DB-2222",
+        toStationId: "place-sstat",
+        day: "weekday",
+        time: 51300,
+    };
+
+    const journeyParams3: JourneyParams = {
+        fromStationId: "place-KB-0351",
+        toStationId: "place-PB-0356",
+        day: "weekday",
+        time: 73358,
+    };
+
+    const journey = [journeyParams1, journeyParams2, journeyParams3];
```

Comment:
This array (and its elements) should be taken out of the body of the component, and moved towards the top of the file (I'd say right beneath `scenarioIds`). One reason is that this code gets re-run every time the component renders, but its result never changes — this wastes a little bit of time, but for small objects like this it's pretty insignificant.

The more important reason is readability. This is a bit subjective, but in my opinion adding code to a function body signals to the reader that it is going to compute something based on the inputs to the function, which is not the case here.